CONTRIBUTING.md

```diff
@@ -104,21 +104,37 @@ If you want to edit an existing docstring signature, you **first** have to chang
 edit the helpdb.jl or inline method docstrings.  The existing signatures in the `doc/stdlib/*.rst` files are pattern matched to base docstrings and the new content overwrites the content in `doc/stdlib/`.
 The signature definitions **must** be in sync or else the pattern match will fail and documentation will be lost in the result.
 To add entirely new methods to the `stdlib` documentation, first add the signature in the appropriate `doc/stdlib/*.rst` file before writing the docstring, rebuilding Julia, and re-running `doc/genstdlib.jl`.
-Pattern matching requires that multiline method signatures' inter-line character alignment in `doc/stdlib/*.rst` match that in the corresponding docstring. For example, docstring
+Pattern matching requires that multiline method signatures' inter-line character alignment in `doc/stdlib/*.rst` match that in the corresponding docstring. In the following example,
+where `␣` represents a single space and not an actual `␣` character, docstring
+
 ```julia
 """
-    foo(bar, baz,
-        qux, quux)
+␣␣␣␣foo(bar, baz,
+␣␣␣␣␣␣␣␣qux, quux)
 
 Foo `bar`, `baz`, `qux`, and `quux`.
 """
 ```
-will only match entries in `doc/stdlib/*.rst` of the form
-```rst
-.. function:: foo(bar, baz,
-..............    qux, quux)
+
+will only match entries in `doc/stdlib/*.rst` beginning with
+
 ```
-where `.` in the preceding snippet's second line is wildcard, and their number preceding indent-`    qux, quux)` matches the character count of `.. function:: `.
+..␣function::␣foo(bar, baz,
+␣␣␣␣␣␣␣␣␣␣␣␣␣␣␣␣␣␣qux, quux)
```

Comment:
That's unfortunate. How does `⋅` (`\cdot`) look:

```
..⋅function::⋅foo(bar, baz,
⋅⋅⋅⋅⋅⋅⋅⋅⋅⋅⋅⋅⋅⋅⋅⋅⋅⋅qux, quux)
```

?

Otherwise will probably just go back to `.` (fullstop), but note that it just represents a space, not a literal `.`.